.github/workflows/scout-pr.yml

```
@@ -0,0 +1,71 @@
+name: Container security (Docker Scout)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)